roadmap.md

```diff
@@ -4,154 +4,95 @@ This document defines a high level roadmap for KubeEdge development.
 
 The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) represent the most up-to-date plans.
 
-The roadmap below outlines KubeEdge’s 2023 feature plan.
+The roadmap below outlines KubeEdge’s 2024 feature plan.
 
-## 2023 H1 
+## SIG Node
 
-### SIG Node
+- Continuous follow up Kubernetes release
+- Support edge nodes running on mac OS and RTOS 
```

Comment:
by `RTOS` here, what actual platform are we looking at? like 1st target? if there is already dedicated tracking issue on github, i would love to track that for sure.

Reply:
> by `RTOS` here, what actual platform are we looking at? like 1st target? if there is already dedicated tracking issue on github, i would love to track that for sure.

https://github.com/kubeedge/kubeedge/issues/4858    This issue is still in research phase.